src/rust/engine/process_executor/src/main.rs

```diff
@@ -248,7 +248,10 @@ async fn main() {
       local_only_store.into_with_remote(
         cas_server,
         args.remote_instance_name.clone(),
-        root_ca_certs,
+        grpc_util::TlsConfig {
```

Comment:
Use constructor? Although this code is better since it makes use of `TlsConfig::default`.